types/cesium/index.d.ts

```diff
@@ -4270,7 +4270,7 @@ declare namespace Cesium {
             credit?: Credit | string;
         });
         static fromGltf(options: {
-            url: string;
+            url: Resource | string;
             basePath?: string;
```

Comment:
```suggestion
            basePath?: Resource | string;
```
mind amending this for same reason?
Also please add a simple test, thx!